### DISCUSSION TITLE
Document TMC Autotune error solution in Wiki

### DISCUSSION THREAD

**Discussion by fzechert** [Guilouz/Klipper-Flsun-Speeder-Pad#134] [General]
Hi all :wave: 

I used the instruction in this wiki page https://github.com/Guilouz/Klipper-Flsun-Speeder-Pad/wiki/Use-Klipper-TMC-Autotune, respectively in the repository https://github.com/andrewmcgr/klipper_tmc_autotune to install Klipper TMC Autotune on the V400.

After the installation, the printer did not start successfully anymore with the following error:

```txt
Unhandled exception during connect
Traceback (most recent call last):
  File "/home/pi/klipper/klippy/klippy.py", line 175, in _connect
    self._read_config()
  File "/home/pi/klipper/klippy/klippy.py", line 141, in _read_config
    self.load_object(config, section_config.get_name(), None)
  File "/home/pi/klipper/klippy/klippy.py", line 121, in load_object
    mod = importlib.import_module('extras.' + module_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/pi/klipper/klippy/extras/autotune_tmc.py", line 2, in <module>
    from enum import Enum
ImportError: No module named enum
```

The solution to this problem is to install the python module `enum` with the following commands

```
# ssh to your printer
source ~/klippy-env/bin/activate
pip install enum
deactivate
```

Is this worth adding to the wiki page?
I would have proposed a pull request, but I don't know on how to contribute to a Wiki page.

For reference, see: https://github.com/andrewmcgr/klipper_tmc_autotune/issues/40

Best regards

**Comment by h3rm** [comment]
@fzechert i got the same error 
1. while installing autotune 
AND 
2. (again) today after installing the adxl -> i had problems with numpy and had to reinstall it with the guide "If you have an error when installing Numpy, follow this steps, if not continue:" @ wiki https://github.com/Guilouz/Klipper-Flsun-Speeder-Pad/wiki/Measure-resonances-with-Fysetc-Portable-Input-Shaper )..

My solution for the adxl:
-After the last step "MEASURE_AXES_NOISE" -> i got an error  in the moonraker console (ACCELEROMETER_QUERY was ok/i got the output)
-After that i did the normal numphy install again. After the restart and updating the .confs -> "failed to import `numpy` module", 
-After the "If you have an error when installing Numpy, follow this steps, if not continue:" (wiki) reinstall of numphy -> "No module named enum" 
-then i used your code
-[update] worked for me and no errors any more 👍 
14:48:34 Axes noise for xy-axis accelerometer: 11.893819 (x), 22.246238 (y), 13.559724 (z)
14:48:31 MEASURE_AXES_NOISE
14:48:13 accelerometer values (x, y, z): 222.061783, -9193.145976, 370.102971
14:48:11 ACCELEROMETER_QUERY

big thx for your post!
